.github/workflows/pull-request.yml

```diff
@@ -47,6 +47,7 @@ jobs:
     - name: Run e2e tests
       env:
         JWT_SECRET: "secret"
+        WEB3_URL: 'https://volta-rpc.energyweb.org/'
```

Comment:
is this required? it says in the readme that it's optional

Reply:
no it wasn't. removed it.